### ISSUE TITLE
Using gnomAD PON for Tumor-Only Analysis Pipeline

### ISSUE THREAD

**Issue by ntbeileh** [lima1/PureCN#185]
I was interested in using the gnomAD panel of normals to run the tumor-only analysis pipeline described here: https://bcbio-nextgen.readthedocs.io/en/latest/contents/purecn.html#create-project-structure

The pon vcf is found here https://console.cloud.google.com/storage/browser/gatk-best-practices/somatic-hg38?pageState=(%22StorageObjectListTable%22:(%22f%22:%22%255B%255D%22))&prefix=&forceOnObjectsSortingFiltering=false

However going through the workflow, in describing the creation of the panel of normals, through that process bcbio produces mapping_bias_hg38.rds and normalDB_hg38.rds which are used in the project structure of the tumor-only analysis.

In order to produce those files, the workflow says to use the BAM and BED files used to create that PON but I do not have those files. It is not possible to extract a list of samples either from that VCF file.

My question was, given these constraints is it possible to use the gnomAD panel of normals for PureCN's tumor-only analysis workflow?

**Comment by lima1** [comment]
We use the pon for coverage normalization (that’s the normalDB_hg38.rds) and for finding biases in mapping of alt alleles (when alt reads have a lower chance of making it in the VCF). For coverage normalization, you need a few process matched normals. For mapping bias, as long as a very similar pipeline was used, and I guess read lengths and insert size distributions are not too different, samples from other assays can be added. 

GnomAD annotation is supported in Mutect2 VCFs, but for mapping bias we would need the raw multi-sample VCF with all alt and ref read counts.

So TL;DR, unfortunately not possible.

**Comment by ntbeileh** [comment]
Thank you for your response!
As a follow up question, to get my panel.bed file can I just convert my multi-sample vcf file to a bed file?

**Comment by lima1** [comment]
Should work, yes. Filter for low coverage though. Then the script that generates the final interval file should resize and merge to something that’s not too different from the true baits coordinates. I would still try to get the real file from the lab that sequenced the samples. Can you share what assay you’re looking at?

**Comment by lima1** [comment]
Closing now since the original question is answered, feel free to follow up here or open new issues.

**Comment by ntbeileh** [comment]
Sorry to get back to you so late. Unfortunately I can't share my data. However I do have the original BAM files from the lab. Should the panel.bed file also come from the company who sequenced them as well? I was originally going forward with converting these bam files to bed format.

**Comment by lima1** [comment]
If it’s a proprietary assay like Foundation One, they probably won’t share. But for standard assays sold as kit, yes, the company should provide them.
